File: 09-CustomDialog-CS/MainForm.cs
using System;
using System.Windows.Forms;
using CustomDialog.Dialogs;
using CustomDialog.Dialogs.StyleBuilder;

namespace CustomDialog;

public partial class MainForm : Form
{
    public MainForm()
    {
        InitializeComponent();
        // Start maximized so the full editor toolbar is visible at first run.
        this.WindowState = FormWindowState.Maximized;
    }

    private void MainForm_Load(object sender, EventArgs e)
    {
        winFormHtmlEditor1.Dialog.ImageDialog = new ImageDialog();
        winFormHtmlEditor1.Dialog.HyperlinkDialog = new HyperLinkDialog();
        winFormHtmlEditor1.Dialog.SpellCheckerDialog = new SpellCheckerDialog { Options = winFormHtmlEditor1.SpellCheckOptions };
        winFormHtmlEditor1.Dialog.TableCellDialog = new TableCellPropertiesDialog();
        winFormHtmlEditor1.Dialog.TableDialog = new TablePropertiesDialog(winFormHtmlEditor1.Dialog.TableCellDialog);
        winFormHtmlEditor1.Dialog.SymbolDialog = new SymbolDialog();
        winFormHtmlEditor1.Dialog.SearchDialog = new SearchWindow();
        winFormHtmlEditor1.Dialog.StyleBuilderDialog = new WinStyleBuilder();
        winFormHtmlEditor1.Dialog.YouTubeVideoInsertDialog = new YouTubeVideoInsertDialog();
    }
}

File: 09-CustomDialog-VB/MainForm.vb
Imports System
Imports System.Windows.Forms
Imports CustomDialog.Dialogs
Imports CustomDialog.Dialogs.StyleBuilder

Namespace Global.CustomDialog

    Partial Public Class MainForm
        Inherits Form

        Public Sub New()
            InitializeComponent()
            ' Start maximized so the full editor toolbar is visible at first run.
            Me.WindowState = FormWindowState.Maximized
        End Sub

        Private Sub MainForm_Load(sender As Object, e As EventArgs) Handles MyBase.Load
            winFormHtmlEditor1.Dialog.ImageDialog = New ImageDialog()
            winFormHtmlEditor1.Dialog.HyperlinkDialog = New HyperLinkDialog()
            winFormHtmlEditor1.Dialog.SpellCheckerDialog = New SpellCheckerDialog With {.Options = winFormHtmlEditor1.SpellCheckOptions}
            winFormHtmlEditor1.Dialog.TableCellDialog = New TableCellPropertiesDialog()
            winFormHtmlEditor1.Dialog.TableDialog = New TablePropertiesDialog(winFormHtmlEditor1.Dialog.TableCellDialog)
            winFormHtmlEditor1.Dialog.SymbolDialog = New SymbolDialog()
            winFormHtmlEditor1.Dialog.SearchDialog = New SearchWindow()
            winFormHtmlEditor1.Dialog.StyleBuilderDialog = New WinStyleBuilder()
            winFormHtmlEditor1.Dialog.YouTubeVideoInsertDialog = New YouTubeVideoInsertDialog()
        End Sub
    End Class

End Namespace
